packages/relayer/cmd/flags/common.go

```diff
@@ -126,6 +126,13 @@ var (
 		Value:    10,
 		EnvVars:  []string{"ETH_CLIENT_TIMEOUT"},
 	}
+	CORSOrigins = &cli.StringFlag{
```

Comment:
im confused why we need this, we have it as an indexer flag already, and it shouldnt be a flag for processor, so it shouldnt be common?

Reply:
I saw it listed under commonCategory flags already and processor config did already have CORSOrigins as a flag, I could remove it and make it an indexer only flag?